### ISSUE TITLE
protonet_miniimagenet.py results do not match the reported results

### ISSUE THREAD

**Issue by chenxianhong** [learnables/learn2learn#249]
Hi, for the examples/vision/protonet_miniimagenet.py, I have two puzzles. 
First, in the examples/vision/README.md, it is said that 
"
For 5 shot 5 ways:

~~~shell
python examples/vision/protonet_miniimagenet.py --shot 5 --train-way 20
~~~
"
Why the train_way is set to be 20 here, not 5 for 5 ways?

Second, I ran the code 'python examples/vision/protonet_miniimagenet.py --shot 5 --train-way 20 --gpu 1', but only got the results 'batch 2000: 42.84(42.67)'.  The accuracy in the README.md is 66.5%. 
I didn't change any parameters. Some parameters are list as follows:

    parser.add_argument('--max-epoch', type=int, default=250)
    parser.add_argument('--test-way', type=int, default=5)
    parser.add_argument('--test-shot', type=int, default=1)
    parser.add_argument('--test-query', type=int, default=30)
    parser.add_argument('--train-query', type=int, default=15)

    optimizer = torch.optim.Adam(model.parameters(), lr=0.001)
    lr_scheduler = torch.optim.lr_scheduler.StepLR(
        optimizer, step_size=20, gamma=0.5)

class Convnet(nn.Module):
    def __init__(self, x_dim=3, hid_dim=64, z_dim=64):
        super().__init__()
        self.encoder = l2l.vision.models.ConvBase(output_size=z_dim,
                                                  hidden=hid_dim,
                                                  channels=x_dim,
                                                  max_pool=True)
        self.out_channels = 1600

    def forward(self, x):
        x = self.encoder(x)
        return x.view(x.size(0), -1)

I also ran the code 'python examples/vision/protonet_miniimagenet.py --shot 5 --train-way 5 --gpu 1' and got the results 'batch 2000: 43.48(48.00)'. I do know why the accuracy does not match the reported accuracy?

**Comment by seba-1511** [comment]
Hello @chenxianhong ,

66% is for 5-ways 5-shots but it seems you're testing 5-ways 1-shot. Still, you should be seeing results closer to 49%. What happens if you run `python examples/vision/protonet_miniimagenet.py`, do you get closer to 49%?

 If you wanted to test 5-ways 5-shots, you need to change the values of `--test-way` and `--test-shot` accordingly in your original command.

**Comment by chenxianhong** [comment]
Thank you for the reply @seba-1511 . Yes, if I run python examples/vision/protonet_miniimagenet.py, I can get the result close to 49%. I found that when testing 5-ways 5-shots, I didn't change the value of --test-shot. It was still 1.  If change it to 5, I can get the result close to 66%. 
Thank you very much for the help.
